pkg/commands/container.go

```diff
@@ -248,7 +248,11 @@ type ContainerCliStat struct {
 
 // GetDisplayStrings returns the dispaly string of Container
 func (c *Container) GetDisplayStrings(isFocused bool) []string {
-	return []string{c.GetDisplayStatus(), c.Name, c.GetDisplayCPUPerc()}
+	image := c.Container.Image
```

Comment:
we can just do `image := strings.TrimPrefix(c.Container.Image, "sha256:")` here

Reply:
![2019-08-06 19-01-05屏幕截图](https://user-images.githubusercontent.com/48538419/62534858-95c9da00-b87c-11e9-8332-b7c99f9d2f9e.png)

Since the image name may remain too long, I have just reserved 14 characters here.

Reply:
![2019-08-06 19-08-00屏幕截图](https://user-images.githubusercontent.com/48538419/62535276-88f9b600-b87d-11e9-8057-315f15910ca5.png)

Compare with this one which is more concise and won't take much spaces.

Reply:
I personally don't mind if the image name is long, given we don't yet have any other columns to the right, and the magenta color means it's not too jarring on the eyes. But if you want to shorten it, I would add an ellipsis (`...`) after the 14 characters to communicate to the user that there's more of the image name not being shown

Reply:
I think long name is not very good. But I do think that add an ellipsis after the 14 characters is better than before.  I'm going to do it.

Reply:
If I do that,  it would be inconsistent with those named containers whose name is very long and with the other colums such as container name.

It may not be that bad to remain the whole name since there's no colums to the right now.

Though, I think it could be adjusted in the future when  adding  toggle-able expanded side panel view.

Reply:
So I'll adopt your suggestions that remain the whole name after `sha256:`

Reply:
@jesseduffield all is done. thank you.